src/Glpi/Migration/AbstractPluginMigration.php

```diff
@@ -179,6 +179,28 @@ final protected function checkDbFieldsExists(array $required_fields): bool
         return true;
     }
 
+    /**
+     * Count records from a table with optional conditions
+     *
+     * @param string $table Table name
+     * @param array $conditions Optional WHERE conditions
+     *
+     * @return int Count of records
+     */
+    protected function countRecords(string $table, array $conditions = []): int
```

Comment:
Please add a small test for this method. There is already a test class for this abstract class, adding this test will be simple.